Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -724,6 +752,24 @@ internal ulong GetDirtyMask(bool initialState)
             return dirtyComponentsMask;
         }
 
+        // a mask that contains all the components with SyncMode.Observers
+        internal ulong GetSyncModeObserversMask()
```

Comment:
Add unit test for this method.   See this commit for an example of how to do something very similar:

https://github.com/vis2k/Mirror/pull/1021/commits/d6cf0f16409c2dec57298c00c31c75c61f245fbf

Reply:
done